src/main/scala/io/archivesunleashed/spark/matchbox/WriteGEXF.scala

```diff
@@ -54,14 +54,17 @@ object WriteGEXF {
       "      <node id=\"" +
       r._1._3 + "\" label=\"" +
       r._1._3 + "\" />")).distinct.collect
-    outFile.write("""<?xml version="1.0" encoding="UTF-8"?>
-      <gexf xmlns="http://www.gexf.net/1.2draft" version="1.2">
+    outFile.write("""<?xml version="1.0" encoding="UTF-16"?>
```

Comment:
Recommend changing back to UTF-8, as my vanilla out-of-the-box Gephi chokes on UTF-16.